classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    student.classrom = self
+    @student << student
+  end
+end
```

Comment:
- [ ] It's great that you have added students to the classroom 👍 but it would be nice if you add the classroom in the student as well to complete this one-to-many relationship 😊 You can do it by adding the following code in the `students.rb` file 👍

```ruby
  attr_reader :classroom

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```

Hint: You can see this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) to get an idea on both ways relationship 👍